src/Canvas.js

```diff
@@ -23,11 +23,13 @@ class Canvas {
    * @param {Stream} [options.stream=process.stdout] Writable stream
    * @param {Number} [options.width=stream.columns] Number of columns (width)
    * @param {Number} [options.height=stream.rows] Number of rows (height)
+   * @param {Object} [options.colorSupport] Object with color support
```

Comment:
I don't like the idea of passing the object, it could better to pass a simple string that represents the color mode